x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/jobs_tab_content.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import type { FC } from 'react';
+import React from 'react';
+import {
+  EuiAccordion,
+  EuiCodeBlock,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import type { Module, ModuleJob, ModuleDatafeed } from '../../../../common/types/modules';
+
+interface Props {
+  module: Module;
+}
+
+export const JobsTabContent: FC<Props> = ({ module }) => {
+  const jobConfigsWithDatafeeds = useMemo(
+    () =>
+      module.datafeeds.map(({ job_id: jobId, config }) => {
+        const job = module.jobs.find((j) => j.id === jobId)!;
+        return {
+          ...job,
+          config: { ...job.config, datafeed_config: config },
+        } as ModuleJob & { datafeed_config: ModuleDatafeed['config'] };
+      }),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
module.id is all that is needed and, arguably, quicker since it's a primitive but in this case it doesn't really offer a huge optimization so will remove 👍